From c18473932b33f0df3125e4725304025d2d27c1e2 Mon Sep 17 00:00:00 2001
From: Murala S Swamy <ssmurala@habana.ai>
Date: Mon, 23 Sep 2024 12:48:40 +0300
Subject: [PATCH] [FSW-7256] fs1: declare range register sets

There are total 16 sets of lbw range registers
are there for each initiator, hence added index
enum to use all those sets.

Change-Id: If699c2d4a008e104fe3ae5838c38143c0cdf9775
---

diff --git a/arch/fs1/arc/dev_security.h b/arch/fs1/arc/dev_security.h
index 65f4997..23bde1e 100644
--- a/arch/fs1/arc/dev_security.h
+++ b/arch/fs1/arc/dev_security.h
@@ -40,6 +40,16 @@
 	SEC_ACCESS_CTRL_MAX,
 };
 
+/* In FS1 total 16 sets of LBW RR's are there */
+enum security_lbw_rr_sets {
+	LBW_RR_IDX_0 = 0,
+	LBW_RR_IDX_1 = 1,
+	LBW_RR_IDX_2 = 2,
+	LBW_RR_IDX_3 = 3,
+	LBW_RR_IDX_15 = 15,
+	LBW_RR_INVALID_IDX = 16,
+};
+
 void security_load_revoke_bits(void);
 int security_efuse_image_validation_en(void);
 void boot_rom_eom_overrides(void);
